src/topp/FeatureFinderIdentification.cpp

```diff
@@ -427,6 +430,8 @@ class TOPPFeatureFinderIdentification :
       {
         RTRegion region;
         region.start = *rt_it - rt_tolerance;
+        // TODO
+        // cppcheck-suppress uninitStructMember
```

Comment:
@hendrikweisser can you add this? potentially in a separate issue?

Reply:
is this potentially a false positive as the end should always be set at line 437?